src/components/GlobalStyles.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */
+
+import styled from 'styled-components'
+
+export const Heading1 = styled.h1`
+
+color: red;
+background-color: blue;
+`
```

Comment:
This is one damn skinny global styles !! :-D could be omitted or a lot more code added here

Reply:
Hahaha, I totally agree! In fact, I don't use this one, just forgot to do the clean up...